src/main/java/eu/bbmri_eric/negotiator/api/controller/v3/NetworkController.java

```diff
@@ -0,0 +1,98 @@
+package eu.bbmri_eric.negotiator.api.controller.v3;
+
+import eu.bbmri_eric.negotiator.dto.NetworkDTO;
+import eu.bbmri_eric.negotiator.dto.negotiation.NegotiationDTO;
+import eu.bbmri_eric.negotiator.dto.person.ResourceResponseModel;
+import eu.bbmri_eric.negotiator.dto.person.UserResponseModel;
+import eu.bbmri_eric.negotiator.mappers.NegotiationModelAssembler;
+import eu.bbmri_eric.negotiator.mappers.NetworkModelAssembler;
+import eu.bbmri_eric.negotiator.mappers.ResourceModelAssembler;
+import eu.bbmri_eric.negotiator.service.NegotiationService;
+import eu.bbmri_eric.negotiator.service.NetworkService;
+import eu.bbmri_eric.negotiator.service.PersonService;
+import eu.bbmri_eric.negotiator.service.ResourceService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v3")
+@Tag(name = "Networks", description = "Retrieve connected networks")
```

Comment:
It should not be only retrieval, Please add endpoints for delete, update as well